src/utils/formatDecimals.ts

```diff
@@ -0,0 +1,8 @@
+export function formatDecimal(number: number) {
+  if (!number) {
+    return;
+  }
+  return Number.isInteger(number)
```

Comment:
what is the goal of that helper?

Reply:
It was meant to be a fallback if we run into numbers with multiple decimals.

Reply:
I have removed it again